### ISSUE TITLE
v0.2.5 broken layout

### ISSUE THREAD

**Issue by gerroon** [spikecodes/libreddit#28]
Please see the image to see how the layout looks 

https://i.imgur.com/OR3x11b.png

I am using Docker on Debian Testing with 

`docker run -d --name libreddit -p 8081:8080 spikecodes/libreddit`

**Comment by spikecodes** [comment]
What browser is this issue shown in?

**Comment by gerroon** [comment]
I tried it Firefox (84) and Chromium  on Linux.

Bear in mind that previous version was working perfectly fine.

**Comment by spikecodes** [comment]
Thank you for the bug report. I am not able to reproduce your issue when running from source or pulling from Docker. Are you able to check the browser console logs and send a screenshot of those here?

**Comment by gerroon** [comment]
this is the onyl thing I see

```
Content Security Policy: The page’s settings blocked the loading of a resource at inline (“default-src”). index.js:3366:22

Layout was forced before the page was fully loaded. If stylesheets are not yet loaded this may cause a flash of unstyled content

```

**Comment by spikecodes** [comment]
Those logs don't look too out of the ordinary but the inline styles shouldn't be blocked. Are you able to visit http://**\<your instance url\>**/style.css in your browser? The screenshot you included looks like there is an issue with the styling.

**Comment by gerroon** [comment]
Yes I can access the file alright

**Comment by gerroon** [comment]
It seems like there is something else, the images are broken as you could see in the screenshot, when I try to load an image directly

Not sure why it is trying to load ssl here, since the site is accessed over http like `http://192.168.1.100:8081`

https://192.168.1.100:8081/proxy/aHR0cHM6Ly9iLnRodW1icy5yZWRkaXRtZWRpYS5jb20vU2hRUlZyd0k3VXN5Y0dfZDlsYWhYVk9PWi1MTjZ6T0NaZ1B4NVV3bGxKdy5qcGc=

```

Secure Connection Failed

An error occurred during a connection to 192.168.1.100:8081. SSL received a record that exceeded the maximum permissible length.

Error code: SSL_ERROR_RX_RECORD_TOO_LONG

    The page you are trying to view cannot be shown because the authenticity of the received data could not be verified.
    Please contact the website owners to inform them of this problem.

```

**Comment by spikecodes** [comment]
I think I found the issue! I made an update to the Content Security Policy to "upgrade-insecure-requests" but that might be causing TLS issues for people hosting their own instance without an SSL. Should be fixed with f819ad2.
